CONTRIBUTING.md

```diff
@@ -73,6 +73,7 @@ Our current goal is to publish one minor release every 6 months. Next, we will o
 Although we greatly appreciate any and all contributions to the project, there are a few things that you should take into consideration:
 
 - The MSTG should not be used as a platform for advertisement of commercial tools, companies or individuals. Write-ups should be written with free and open-source tools in mind and commercial tools are typically not accepted, unless as a reference in the security tools section.
+- The MSTG should not be used as a blogging platform: write-ups in the **write-up folder** should be "new" with differnet techniques to crack a crackme, if information is already in the guide: do not duplicate it with additional blogs within the repository.
```

Comment:
I thought we were removing this because we put it as part of the contributing. So no need to include this line.
```suggestion
```